### PR TITLE
#48: Menu highlight

### PR DESCRIPTION
**Pull request by AJV009**
- Active Menu Item Highlight as mentioned in https://github.com/JayKandari/bookhive/issues/48
- Formatted css a bit (https://github.com/JayKandari/bookhive/issues/27)

### CODE REVIEW

**Comment by AJV009** [comment]
Outputs ->
img1

img2

Here my mouse pointer is on the contact option
img3

@rahulsavaria  @shriaas2898 @pragzii3896

**Comment by shriaas2898** [comment]
That's good but I think you could have kept the bgcolor same as the hover color that would look better, just a suggestion.

File: resources/template/menu.php [hunk 1]
```diff
@@ -30,6 +30,7 @@ public function render_header()
 
         echo " <input type='checkbox' id='check'>
             <!--header area start-->
+            <meta name='viewport' content='width=device-width, initial-scale=1'>
```
Comment (shriaas2898) [inline resources/template/menu.php]: Its better to keep `<meta>` tag inside `<head>`, and I have already added it wherever it as missing. You can remove it from here.

**Review by shriaas2898 — CHANGES_REQUESTED** [review]

**Comment by pragatikanade** [comment]
@AJV009 it looks good.. and for hover color same as @shriaas2898 said :+1:

File: resources/template/menu.php (hunk 1 above)
Comment (AJV009) [reply resources/template/menu.php]: oo right i forgot
about it

**Comment by AJV009** [comment]
> That's good but I think you could have kept the bgcolor same as the hover color that would look better, just a suggestion.

I have a counter suggestion ;)
If both colors go same it will be quite difficult to differentiate when someone keeps a mouse pointer on it, what do you think?
@shriaas2898  @pragzii3896

**Comment by shriaas2898** [comment]
> > That's good but I think you could have kept the bgcolor same as the hover color that would look better, just a suggestion.
> 
> I have a counter suggestion ;)
> If both colors go same it will be quite difficult to differentiate when someone keeps a mouse pointer on it, what do you think?
> @shriaas2898 @pragzii3896

That can be differentiated by hovering, the white one just looks really off the theme for me :/ @pragzii3896  and @rahulsavaria can give better suggestions I guess.

**Comment by AJV009** [comment]
> > > That's good but I think you could have kept the bgcolor same as the hover color that would look better, just a suggestion.
> > 
> > 
> > I have a counter suggestion ;)
> > If both colors go same it will be quite difficult to differentiate when someone keeps a mouse pointer on it, what do you think?
> > @shriaas2898 @pragzii3896
> 
> That can be differentiated by hovering, the white one just looks really off the theme for me :/ @pragzii3896 and @rahulsavaria can give better suggestions I guess.

Ooooookay 🙈️😂️😂️😂️😂️😂️ my design sense is still bad I guess.
Why am I always like this! Am I color blind! 😭️😭️😭️
😂️😂️😂️
NVM I'll change the colors

**Comment by pragatikanade** [comment]
@AJV009  Hey if you want to differentiate.. you can select other colour but.. don't go with white :+1:

**Comment by AJV009** [comment]
img1

@pragzii3896 @shriaas2898 @rahulsavaria 

Maybe we should have followed a yellowish color, because hive is yellow in color now it looks like hive fell into water 😂️😂️😂️

**Comment by guptahemant** [comment]
@AJV009 please resolve conflicts

**Comment by AJV009** [comment]
@guptahemant Hi!i!i!i!
Conflicts resolved

**Review by guptahemant — APPROVED** [review]

**Pull request merged**